packages/data/json/9.10.14.json

```
@@ -614,7 +616,7 @@
       "questionText": "Is this compartment <defined-term>sprinklered<defined-term> per <pdf-download-link>Article 3.2.5.12.</pdf-download-link>?",
```

```suggestion
      "questionText": "Is this compartment <defined-term>sprinklered</defined-term> per <pdf-download-link>Article 3.2.5.12.</pdf-download-link>?",
```
This html typo bombs the parser

---

Ahh yeah. I'll have that fixed in the next PR. This PR is only for questions up to P6.